boards/earlgrey-nexysvideo/src/io.rs

```diff
@@ -56,3 +61,14 @@ pub unsafe extern "C" fn panic_fmt(pi: &PanicInfo) -> ! {
         &CHIP,
     )
 }
+
+#[cfg(test)]
+#[no_mangle]
+#[panic_handler]
+pub unsafe extern "C" fn panic_fmt(pi: &PanicInfo) -> ! {
+    let writer = &mut WRITER;
+
+    let _ = write!(writer, "{}", pi);
+    // Exit QEMU with a return code of 1
+    rv32i::semihost_command(0x18, 1, 0);
```

Comment:
AFAIK earlgrey-nexysvideo also runs on an FPGA board. How does this behave here?